core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2079,6 +2079,14 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
               .build();
+  public static final PropertyKey MASTER_WORKER_INFO_CACHE_REFRESH_TIME =
+      new Builder(Name.MASTER_WORKER_INFO_CACHE_REFRESH_TIME)
+          .setDefaultValue("10sec")
+          .setDescription("The worker information list will be refreshed "
+              + "after being cached for this time period. ")
```

Comment:
```suggestion
              + "after being cached for this time period.")
```

Also, can you add what the implications may be if it is too short, and too long?

Reply:
DONE